modules/installation-osp-download-image-restricted.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer-restricted.adoc
+
+[id="installation-osp-download-image-restricted_{context}"]
+= Retrieving a {op-system} image for restricted installation
+
+Download the {op-system-first} image to install {product-title} on a restricted-network {rh-openstack-first} environment.
+
+.Prerequisites
+
+* Obtain the {product-title} installation program. For a restricted network installation, the program is on your bastion host.
+
+.Procedure
+
+// Is this procedure going to run afoul of Support guidelines?
```

Comment:
@morenod I'd anticipate Support pushback on using the procedure as described here. Could this be replaced easily by directing the reader to go to https://access.redhat.com/downloads/content/290/ver=4.4/rhel---8/4.4.3/x86_64/product-software (or something close) to download the image?

Reply:
yes, I think we can use official sites for downloading the RHCOS image. once a release is published, if image is uploaded to access.redhat.com, we can use it